### DISCUSSION TITLE
Putty access denying

### DISCUSSION THREAD

**Discussion by Shravang007** [learndevopsonline/learndevopsonline#901] [Q&A] [answered]
Putty denying my access while entering password. how can i access putty ?
![Screenshot 2023-05-21 093405](https://github.com/learndevopsonline/learndevopsonline/assets/133180652/94b04bcb-b73e-4b99-861e-29a85ac62c75)

**Comment by rpraveenkumar1220** [comment, marked as answer]
login as :  centos
Password : DevOps321

Don't use any ip address after centos
